RecoLocalCalo/HGCalRecProducers/test/testHGCalRecoLocal_cfg.py

```diff
@@ -1,7 +1,7 @@
 import FWCore.ParameterSet.Config as cms
 from Configuration.StandardSequences.Eras import eras
 
-process = cms.Process("testHGCalRecoLocal",eras.Phase2C2_timing)
+process = cms.Process("testHGCalRecoLocal",eras.Phase2C2)
```

Comment:
The timing eras now requires FastTiming layers in the scenario. So the unit test does not run as it is. We made a wrong move last time. We had to drop some of the SLHC customization parts which were removed. This unit test without this change would not work